web/src/components/Failure.tsx

```diff
@@ -16,17 +16,18 @@ const Centered = styled.div`
 
 type PropsType = {
   errorMessage: string
-  goToPath?: string
+  goToPath?: string | null
   goToMessage?: string
   t: TFunction<'error'>
 }
 
+// screen reader test
 const Failure = ({ errorMessage, goToPath = '/', goToMessage = 'goTo.start', t }: PropsType): ReactElement => (
   <Centered>
-    <div>{t(errorMessage)}</div>
     <div>
-      <FontAwesomeIcon icon={faFrown} size='5x' />
+      <FontAwesomeIcon icon={faFrown} size='4x' />
     </div>
+    <div role='alert'>{t(errorMessage)}</div>
```

Comment:
What is the reason for this design change? Did you discuss this with our designers?

Reply:
I reused the empty search result layout. It was mainly the same as the Failure layout.
We could do it the other way round also.
The only reason why I used the order of the empty search result was that having the text above the smiley looked strange (with url there is text above and below the smiley)